mobility_sensing/scripts/ClosestWallFinder.py

```diff
@@ -251,5 +251,5 @@ def PassedPlane(self, plane_to_pass, testplanepoints, position):
 
 
 if __name__ == '__main__': #Run Code
-    node = ClosestWallFinder(True, True) #, True)
+    node = ClosestWallFinder(True) #, True) #, True)
```

Comment:
make this  `node = ClosestWallFinder(visualize=True)` for clarity and to clean up commented code.